driver_scripts/driver_grid.cray.sh

```diff
@@ -65,32 +66,32 @@ module list
 # Set grid specs here.
 #-----------------------------------------------------------------------
 
-export gtype=uniform       # 'uniform', 'stretch', 'nest'
-                           # 'regional_gfdl', 'regional_esg'
-
+export gtype=uniform           # 'uniform', 'stretch', 'nest', 
+                               # 'regional_gfdl', 'regional_esg'
 if [ $gtype = uniform ]; then
   export res=96
-  export add_lake=false   # Add lake frac and depth to orography data.
-                          # Uniform grids only.
-  export lake_cutoff=0.20 # lake frac less than lake_cutoff is ignored
+  export add_lake=false        # Add lake frac and depth to orography data.
+  export lake_cutoff=0.20      # lake frac < lake_cutoff ignored when add_lake=T
 elif [ $gtype = stretch ]; then
   export res=96
   export stretch_fac=1.5       # Stretching factor for the grid
   export target_lon=-97.5      # Center longitude of the highest resolution tile
   export target_lat=35.5       # Center latitude of the highest resolution tile
 elif [ $gtype = nest ] || [ $gtype = regional_gfdl ]; then
-  export res=96
+  export add_lake=false        # Add lake frac and depth to orography data.
+  export lake_cutoff=0.20      # lake frac < lake_cutoff ignored when add_lake=T
+  export res=768
```

Comment:
I see you tested with a C768 regional grid.  Will the lake code work with other resolutions?